lib/META

```diff
@@ -1,6 +1,6 @@
 # OASIS_START
-# DO NOT EDIT (digest: 0a469b53195f4c578082ce622a25da7c)
-version = "1.0.1"
+# DO NOT EDIT (digest: 464dc1b5e16adf8099f0fd9ceeaaa775)
```

Comment:
I think there are some more files that should be removed: `lib/fd_send_recv.mllib`, `lib/libfd_send_recv_stubs.clib`.